libs/core/src/lib/input-group/input-group.component.html

```diff
@@ -30,7 +30,7 @@
                 [compact]="compact"
                 [id]="_addOnButtonId"
                 [attr.tabindex]="buttonFocusable ? 0 : -1"
-                [attr.title]="iconTitle ? iconTitle : null"
+                [attr.title]="iconTitle? iconTitle: (glyphAriaLabel ? glyphAriaLabel: glyph)"
```

Comment:
Could be simplified `iconTitle || glyphAriaLabel || glyph`.

Reply:
Done